### PR TITLE
Fix diff roi

### PR DESCRIPTION
**Pull request by sameera2004**
Fixed the functions in the diff_roi_choice.py after changes in the correlation.py

### CODE REVIEW

**Comment by sameera2004** [comment]
@tacaswell I think I have to close the PR #248 Divide the circle : find the indices, number of pixels, pixel indices. I fixed this new PR, I fixed all the functions in this , the test are passing but there is a problem with the coverage,(missed 7 lines)

**Comment by danielballan** [comment]
Suggested name changes:

`detector_size`, `img_shape`, etc. -> `shape`
`roi_data` -> `roi_specs` and remove `num_rois`, which we can infer from the length of `roi_specs`

**Comment by danielballan** [comment]
Scikit-image has fast functions based on published CS literature for drawing rings: http://scikit-image.org/docs/dev/api/skimage.draw.html#skimage.draw.circle

File: skxray/diff_roi_choice.py [hunk 1]
```diff
@@ -116,18 +109,15 @@ def roi_rectangles(num_rois, roi_data, detector_size):
...
-    w = np.where(mesh > 0)
-    grid = np.indices((detector_size[0], detector_size[1]))
-    pixel_list = ((grid[0]*detector_size[1] + grid[1]))[w]
+    roi_labels = mesh[mesh > 0]
 
-    roi_inds = mesh[mesh > 0]
+    roi_labels, indices = extract_label_indices(mesh)
 
-    return mesh.reshape(detector_size), roi_inds, num_pixels, pixel_list
+    return mesh.reshape(detector_size), roi_labels, indices
```
Comment (danielballan) [inline skxray/diff_roi_choice.py]: Only return the first bit, not the second two.

**Comment by tacaswell** [comment]
What is this PR?

**Comment by danielballan** [comment]
Superceded by Sameera's #269, my #272, and a forthcoming PR that Sameera and I are pair-coding currently....

**Pull request closed without merging**